xchange-kucoin/src/main/java/org/knowm/xchange/kucoin/KucoinAdapters.java

```diff
@@ -161,14 +161,20 @@ public static ExchangeMetaData adaptMetadata(
     HashMap<String, CurrencyMetaData> stringCurrencyMetaDataMap = new HashMap<>();
     for (CurrenciesResponse currenciesResponse : list) {
       BigDecimal precision = currenciesResponse.getPrecision();
-      String withdrawalMinFee = currenciesResponse.getWithdrawalMinFee();
-      String withdrawalMinSize = currenciesResponse.getWithdrawalMinSize();
+      BigDecimal withdrawalMinFee = BigDecimal.ZERO;
+      BigDecimal withdrawalMinSize = BigDecimal.ZERO;
```

Comment:
this fix is ok but I think it would be better to treat unknown values as null and pass nulls to  CurrencyMetaData

Reply:
Thx but i'm quite scared to what it could mean "later" with null value...

Reply:
@walec51  you think I should try with:
```
BigDecimal withdrawalMinFee = null;
BigDecimal withdrawalMinSize = null;
```

Reply:
@walec51 ok i changed it to null